src/audio_output_alsa.cc

```diff
@@ -90,12 +92,14 @@ bool AudioOutputALSA::Start() {
 
       std::shared_ptr<std::vector<unsigned char>> data = audioData[0];
       audioData.erase(audioData.begin());
-      int frames = data->size() / 2;  // 1 channel, S16LE, so 2 bytes each frame.
+      int frames =
```

Comment:
Is this the correct format? Would it make more sense to put the comment on another line if it gets broken up like this?

Reply:
Definitely. Fixed.